pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -273,7 +274,10 @@ const SourceRow = ({ source, sourceType, networks, nodeDevices, providerName, on
                     key={LOCAL_INSTALL_MEDIA_SOURCE}>{_("Local Install Media")}</Select.SelectEntry>
                 <Select.SelectEntry data={URL_SOURCE} key={URL_SOURCE}>{_("URL")}</Select.SelectEntry>
                 { providerName == 'LibvirtDBus' &&
-                <Select.SelectEntry data={PXE_SOURCE} key={PXE_SOURCE}>{_("Network Boot (PXE)")}</Select.SelectEntry> }
+                <Select.SelectEntry title={connectionName == 'session' ? _("Network Boot is available only when using System connection") : null}
+                    disabled={connectionName == 'session'}
+                    data={PXE_SOURCE}
+                    key={PXE_SOURCE}>{_("Network Boot (PXE)")}</Select.SelectEntry> }
```

Comment:
Missing line break here? (But not important enough to re-push, as long as eslint is happy)

Reply:
Done